docs/rackhd2.0/install_os_current.rst

```diff
@@ -51,14 +51,14 @@ Check the workflow is active or inactive:
 
 .. code-block:: REST
 
-    curl <server>/api/1.1/nodes/<identifier>/workflows/active
+    curl <server>/api/current/nodes/<identifier>/workflows/active
 
 
 Stop the active workflow to cancel OS installation:
 
 .. code-block:: REST
 
-    curl -X DELETE <server>/api/1.1/nodes/<identifier>/workflows/active
+    curl -X DELETE <server>/api/current/nodes/<identifier>/workflows/active
 
```

Comment:
Similar to /workflows, the way you canel an active workflow on a node is by using the PUT method 
```
node/:id/workflows/action
```
and pass
```javascript
{"command": "cancel"} 
in the body of this PUT now
```